### PR TITLE
FInish PoetListingPage

### PR DESCRIPTION
**Pull request by char0076b900**
Poet page is a listing of poets. It has a search function that searches poets by name. Users can sort the listing by date, views, likes, alphabet in ascending and descending order. Filter by the poets' active year is also supported. 

The body of the page is actual listing of poets.

### CODE REVIEW

File: src/components/PoetListingPage.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  }
+
+  search(e) {
+    e.preventDefault();
+    this.query.q = this.state.q;
+    this.pushHistory();
+  }
+
+  searchYear(e) {
+    // da e.preventDefault();
```
Comment (Kailang) [inline src/components/PoetListingPage.js]: Delete comment.

File: src/components/PoetListingPage.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    /* eslint-disable-next-line no-invalid-this */
```
Comment (Kailang) [inline src/components/PoetListingPage.js]: No need.

File: src/components/PoetListingPage.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    console.log(e.target.value);
```
Comment (Kailang) [inline src/components/PoetListingPage.js]: Remove console log.

File: src/components/PoetListingPage.js (hunk 1 above)
Comment (char0076b900) [reply src/components/PoetListingPage.js]: Done.

File: src/components/PoetListingPage.js (hunk 2 above)
Comment (char0076b900) [reply src/components/PoetListingPage.js]: Done.

File: src/components/PoetListingPage.js (hunk 3 above)
Comment (char0076b900) [reply src/components/PoetListingPage.js]: Removed.

**Comment by Kailang** [comment]
Just remember to remove the useless codes before sending the PR.

**Pull request merged**